command/harness/helper.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func getStreamLogger(cfg leapi.LogConfig, logKey, correlationID string) *lelivelog.Writer {
 	client := lestream.NewHTTPClient(cfg.URL, cfg.AccountID,
-		cfg.Token, cfg.IndirectUpload, false)
+		cfg.Token, cfg.IndirectUpload, cfg.IndirectUploadSetting, false)
```

Comment:
let's make sure that this is merged only when le version bump is updated in this pr